cmd/minikube/cmd/start.go

```diff
@@ -679,6 +679,20 @@ func validateKubernetesVersions(old *cfg.Config) (string, bool) {
 		return nv, isUpgrade
 	}
 
+	oldestVersion, err := semver.Make(strings.TrimPrefix(constants.OldestKubernetesVersion, version.VersionPrefix))
+	if err != nil {
+		exit.WithCode(exit.Data, "Unable to parse oldest Kubernetes version from constants: %v", err)
+	}
+
+	superOldVersionLimit := semver.Version{Major: 1, Minor: 9, Patch: 0}
```

Comment:
Thanks for this PR!

If possible, I would rather not have two effective "oldest" constants. The intent of `constants.OldestKubernetesVersion` was to have a constant that defined the oldest possible version. If v1.9.x works, the variable should reflect that state.

Do you mind updating `constants.OldestKubernetesVersion` to v1.9.0 so that the intent is met and the logic simple? If it happens to fail tests, we can stick with v1.10.x and add a `--force` flag so that users can still bypass the check.

Reply:
Hi Thomas, thanks for the review.

As far as I understand this is the behavior requested in the original issue #4673. According to your input we can remove the check for `superOldVersionLimit` and just return a warning (without exit) to the user if the specified version is less than `constants.OldestKubernetesVersion`. 

Can you check this comment to make things clear about what needs to be done https://github.com/kubernetes/minikube/issues/4673#issuecomment-508633818

Reply:
Hi sorry for the confusion, I think I contributed to the confusion ! my fault !

Thomas is right, rethinking this I believe it would be hard to maintain two separate versions of supported kuberentes. 

the original reason we created this issue was someone was trying to start an oddly old kubernetes by mistake (1.2 instead of 1.12)


here is my current new suggestion based on @tstromberg input:

we need to exit on oldest supported kubernets version ( yes we can update the oldest possible one to 1.9) however we need to infrom the user in the exit message, that to override this check please add "--force"

Reply:
@serhatcetinkaya - Yes, that sounds like the right way to go. I spoke to Medya about this yesterday and he seemed willing to accept it as well. I've added my thoughts to the bug. Thank you!

Reply:
Thanks for the clarification @medyagh @tstromberg, I am working on this, I will submit as soon as I finish.

Reply:
Huzzah! start.go now has support for a --force flag.  If you change the logic to use it - I think we should merge this ASAP. 

```
if viper.GetBool(force) {
  out.WarningT("Kubernetes {{.version}} is not supported by this release of minikube", ...)
} else {
  exit.WithCode(exit.Data, "Sorry, Kubernetes {{.version}} is not supported by this release of minikube")
}
```

Please let me know if you need any help.